testing/himem_test/Kconfig

```diff
@@ -0,0 +1,28 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+config TESTING_HIMEM_TEST
+	tristate "HIMEM TEST APP"
+	---help---
+		Enable the SMP APP
```

Comment:
SMP APP?

Reply:
It has been change to "himem cdev test app"